### ISSUE TITLE
Game seems laggy/stuttery since the new update.

### ISSUE THREAD

**Issue by PainDota** [jeffhill/Dota2#946]
### Description

I don't know if this server specific but I asked other players as well they were lagging and stuttering. For context, playing ranked on sea server.

My PC specs are r5 5600x and 3070ti I don't think its hardware related, pings are fine and I have 0 loss.

### Example Match ID (and possibly Timestamp)

6608890079

**Comment by AeonDota** [comment]
I had this as well. Over 100fps and only 20 ping but heros seem to be stuttering as they moved

**Comment by MrDemonRush** [comment]
Yup, same issue. Constant 100 fps yet heroes move very stuttery.

**Comment by BenjinZA** [comment]
Possible duplicate of #915

**Comment by oicangis** [comment]
Since the new update i has a Lot of problem with dota, the Game is frozen in the laning phase, My PC usually run the Game at 120fps and before the new update i had the Game sometimes at 10fps, PLEASE FIX THIS!!
